libs/aws/src/lib/cloudwatch-logger/is-error.function.ts

```diff
@@ -1,6 +1,7 @@
 import { DataAlreadyAcceptedException, InvalidSequenceTokenException } from '@aws-sdk/client-cloudwatch-logs'
 import { SmithyException } from '@aws-sdk/types/dist/types/shapes'
 
+// eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
 export function isError(value: any | unknown): value is { name: string; message: string } {
```

Comment:
i think it's ok to change the type of value to `unknown` since all error objects work like that now